typescript/sdk/src/HyperlaneApp.ts

```diff
@@ -4,10 +4,9 @@ import {
   connectContracts,
   serializeContracts,
 } from './contracts';
-import { MultiProvider } from './providers/MultiProvider';
+import { MultiProvider } from './providers';
```

Comment:
If you strongly prefer using intermediate index files I'm not super opposed, but I def prefer packages without them. Some downsides:
1. They bloat repos by creating a bunch of files
2. They lead to inconsistent import paths because there are now >1 valid paths to a resource
3. They sometimes reduce the efficacy of tree shaking by bundlers
4. They increase likelyhood of circular dependencies

Reply:
IIRC @yorhodes was also against them but I may be remembering that wrong

Reply:
Yeah I'm happy to drop, felt cleaner to me didn't remember that we'd already discussed